Meden-Agan-27/travel-worms
# Bookshelf feature

What I did:
- see, create and delete my bookshelves
How to test:
migrate and seed
login
http://localhost:3000/bookshelves
you have to see just the curent_user's bookshelves
you can create bookshelves 
you can delete
edit is not implemented

please see also the code and the routes
